crates/types/src/configuration.rs

```diff
@@ -487,10 +487,14 @@ pub struct CameraMatrixParameters {
 
 #[derive(Clone, Debug, Default, Deserialize, Serialize, SerializeHierarchy)]
 pub struct FallProtection {
+    pub ground_impact_angular_threshold: f32,
     pub ground_impact_head_stiffness: f32,
-    pub arm_stiffness: f32,
+    pub ground_impact_body_stiffness: f32,
+    pub time_free_motion_exit: f32,
```

Comment:
Please use `Duration` instead of `f32` for time intervals.

Reply:
Done

Reply:
Please do this for all 'time' parameters.